### PR TITLE
Update examples docs

### PR DESCRIPTION
**Pull request by forsyth2**
Update examples in the documentation.

Once #341 and this PR merge, #311 can be closed.

### CODE REVIEW

**Comment by forsyth2** [comment]
#332 should merge before this PR so that files that should be ignored aren't included.
#341 must merge before this PR so that the code links exist for the documentation to point to.
After this PR merges, work can begin on #299. (Note that versioned documentation PR should be merged before any documentation specific to the next release is added).

File: source/examples.rst [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+First, make sure you're using version 2.0.0 or greater of e3sm_diags.
+
+Then do one of the following:
+
+* Enter a ``conda`` environment that has ``e3sm_diags`` installed.
+   * Either follow `a quickstart guide <quickguides/index.html>`__ or `the instructions here <install.html>`__.
+* If you have the container downloaded, then download `this script <https://raw.githubusercontent.com/E3SM-Project/e3sm_diags/master/acme_diags/container/e3sm_diags_container.py>`__ to run it.
+   * If you're using a container, replace every instance of
+     ``e3sm_diags`` in this guide with ``python e3sm_diags_container.py --<your_container_runtime>``.
+   * See `this guide <quickguides/quick-guide-cori.html>`__ for more information.
```
Comment (forsyth2) [inline source/examples.rst:19]: This seems like it should instead be a link to setting up the container. It currently links to the Cori quick guide. Is there a better link to use or is this what we want here?

**Review by forsyth2 — COMMENTED** [review]
@chengzhuzhang The docs associated with the examples are ready for review. Most changes are in `source/examples.rst`. I also deleted the old `examples` directory.

File: source/examples.rst (hunk 1 above)
Comment (chengzhuzhang) [reply source/examples.rst:19]: Since we are not supporting containers currently. Let's point to the installation guide, instead?

Comment (forsyth2) [reply source/examples.rst:19]: In that case, should I just remove the part about containers entirely?

Comment (chengzhuzhang) [reply source/examples.rst:19]: Yes!

**Comment by forsyth2** [comment]
@chengzhuzhang I removed the part about containers. Let me know if the rest of `source/examples.rst` looks good. If so, I'll merge.

**Review by chengzhuzhang — APPROVED** [review]
Looks good!

**Pull request merged**